src/main/java/tc/oc/pgm/restart/RestartManager.java

```diff
@@ -35,6 +35,8 @@ public static RestartManager get() {
   private Instant queuedAt;
   private String reason;
 
+  private int timedQueuedRestartTaskID;
```

Comment:
This needs a better name, probably `queuedRestartTask`?

Reply:
renamed